acme/dns_challenge.go

```diff
@@ -124,19 +202,20 @@ func unFqdn(name string) string {
 	return name
 }
 
-// waitFor polls the given function 'f', once per second, up to 'timeout' seconds.
-func waitFor(timeout int, f func() (bool, error)) error {
+// waitFor polls the given function 'f', once every 'interval' seconds, up to 'timeout' seconds.
+func waitFor(timeout, interval int, f func() (bool, error)) error {
```

Comment:
I'm just thinking out loud here... but maybe we should add a util.go for stuff like this?